app/views/content/become-a-further-education-teacher.md

```diff
@@ -16,21 +17,27 @@ keywords:
   - NVQ
   - BTEC
   - HND
-  - apprencticeship
 ---
 
-Further education is any study that takes place after secondary school but outside of higher education (e.g. university).
 
-In further education, teaching doesn’t always happen in a classroom. It can take place in community centres, workplaces and private training centres. 
+If you’re interested in teaching young people and adults over 16 who are not studying for a degree, then further education might be for you. 
 
-You could teach a diverse set of learners, including young people and adults.
+## What is further education?
 
-You do not need a degree to start teaching in further education. 
+It covers three key areas: 
```

Comment:
```suggestion
It covers 3 key areas: 
```
sorry to be a pain 🤦🏻‍♀️ but could we just change this cos it's in the GDS style guide https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#numbers and think it's what we try and do elsewhere on site